.Documentation/articles/innoactive-creator/snapzones.md

```diff
@@ -0,0 +1,104 @@
+
+- **Show Highlight Object**: A prefab that contains only the meshes of the preview object.
+- **Shown Highlight Object Color**: The color that the preview object will have.
+
+[![Snap Zone Preview](../images/snap-zones/snap-zone-preview.png "Snap Zone Preview")](../images/snapzones/snap-zone-preview.png)
+
+#### Snap Zone Validation Preview
+
+To provide visual feedback to the trainee when hovering an object near the snap zone, it can be configured as follows:
+
+- **Show Interactable Hover Meshes**: If this option is enabled, the snap zone will show a preview version of the snappable object.
+- **Interactable Hover Mesh Material**: The material that the preview object will have when a valid snappable object interacts with the snap zone.
+
+[![Snap Zone Validation Preview](../images/snap-zones/snap-zone-validation-preview.png "Snap Zone Validation Preview")](../images/snapzones/snap-zone-validation-preview.png)
+
+You can create a snap zone either [automatically](#a-automatically-setup-a-simple-snap-zone) (recommended) or [manually](#b-manually-setup-a-simple-snap-zone).
+
+------
+
+## Automatically setup a simple snap zone
+
+### I. Snappable game object
+
+We need a game object that we want to snap into the snap zone. It needs to have at least one collider component. To make it snappable, we need to add the `Snappable Property` component.
+
+[![Add Snappable Property](../images/snap-zones/box-snappable.png "Add Snappable Property")](../images/snapzones/box-snappable.png)
+
+
+### II. Create snap zone automatically
+
+To automatically create a matching snap zone, click the `Create Snap Zone` button of the `Snappable Property` in the `Inspector` window of the corresponding snappable game object.
+
+[![Create Snap Zone](../images/snap-zones/automatically-create.gif "Create Snap Zone")](../images/snapzones/automatically-create.gif)
+
+> The snap zone is generated in the same position, rotation, and with the same volume as the snappable property object.
+
+
+### III. Adjust snap zone (optional)
+
+You can either change the settings directly in the new snap zone object or change the default settings in the `Snap Zone Settings` window. In order to open the default snap zone settings, **go to the Unity's toolbar** and select: `Innoactive` > `Creator` > `Windows` > `Snap Zone Settings`. All changes are saved automatically, and every snap zone that will be created via the `Create Snap Zone` button afterwards will get the new settings by default. The `Apply settings in current scene` button applies these settings to all snap zones in the current scene even if they were not created automatically.
+
+[![Create Snap Zone](../images/snap-zones/change-settings.gif "Create Snap Zone")](../images/snapzones/change-settings.gif)
+
+
+------
+
+## Manually setup a simple snap zone
+
+In order to properly set up a snap zone we require 3 game objects:
```

Comment:
we need

Reply:
I actually prefer require :)

Reply:
Since apparently more people rather `require`  (this exact same text was already published in the same article for VRTK), I think I´ll leave as it is.